heartbeat/portblock

```diff
@@ -466,8 +466,7 @@ IptablesValidateAll()
 		exit $OCF_ERR_CONFIGURED
 	fi
 	if [ ! -d "$OCF_RESKEY_tickle_dir" ]; then
```

Comment:
You could also get rid of this if statement. mkdir -p will just exit if the directory exists.